### PR TITLE
Revise README for Baseline launch

### PR DESCRIPTION
**Pull request by ddbeck**
Since we're anticipating more activity here soon, I've taken a shot at improving the README, with contributors as a major audience (and using this opportunity to [eat my own dog food](https://github.com/ddbeck/readme-checklist)).

### CODE REVIEW

**Review by dontcallmedom — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
+The project is in its early stages, so we’re still developing tools and processes to help improve and expand our work.
+To get started contributing, take a look the following:
+
+* Learn more about [the WebDX Community Group](https://github.com/web-platform-dx/admin/blob/main/charter.md)
+* Go to [open issues](https://github.com/web-platform-dx/feature-set/issues) to propose new feature groups or report a bug
+* Go to [open PRs](https://github.com/web-platform-dx/feature-set/pulls) to review incoming feature groups and updates
+
+## License
+
+You’re free to TODO: license file needed.
```
Comment (ddbeck) [inline README.md:38]: We ought to have something like this here _and_ a LICENSE file for the whole repo. See also: https://github.com/web-platform-dx/feature-set/pull/163

```suggestion
You are free to use, copy, modify, and distribute `feature-set`, as long as you include the required notices. Read the LICENSE file for details.
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,38 @@
-# Exploring the set of interoperable features in the Web Platform.
+# [`feature-set`](https://github.com/web-platform-dx/feature-set): exploring the set of interoperable features of the web platform
 
-See the [2022 backgrounder](2022-backgrounder.md) for details.
\ No newline at end of file
+By the W3C WebDX Community Group and contributors
+
```
Comment (atopal) [inline README.md:4]: Is this line break intentional?

**Review by atopal — APPROVED** [review]
Thank you Daniel! This looks great.

File: README.md (hunk 2 above)
Comment (ddbeck) [reply README.md:4]: Yes, it looks slightly less cryptic, rendered as HTML (as in https://github.com/ddbeck/feature-set/blob/ad439fc24e0d3acc3321994edd2664c73c16373c/README.md).

Comment (atopal) [reply README.md:4]: 👍

**Pull request merged**